### PR TITLE
Added message component

### PR DESCRIPTION
**Pull request by IrvingJVR-MX**
Added message component
Receiver
image
image
Sender: 
image
image

### CODE REVIEW

File: src/components/Message.tsx [hunk 1]
```diff
@@ -1,15 +1,66 @@
...
+import { useContext } from 'react';
+import { ColorContext } from '../shared/ColorContext';
+
+
+type MessageType = 'sender' | 'receiver';
+
+type MessageProps = {
+  message: string;
+  time: String;
+  type?: MessageType;
```
[the remark this answers is not in the document]
Comment (IrvingJVR-MX) [reply src/components/Message.tsx]: totally agree, changed

File: src/components/Message.tsx [hunk 2]
```diff
@@ -1,15 +1,66 @@
 import { StyleSheet, Text, View } from 'react-native'
 import React from 'react'
+import { useContext } from 'react';
```
[the remark this answers is not in the document]
Comment (IrvingJVR-MX) [reply src/components/Message.tsx]: done

File: src/components/Message.tsx [hunk 3]
```diff
@@ -1,15 +1,66 @@
...
+      textAlign:'justify',
+      marginRight:51
+    },
+    textTime: {
+      color: textColorMessage,
+      fontWeight: '600',
+      fontSize: 12,
+      alignSelf: 'flex-end',
+      marginBottom: -5,
+      marginTop: -14
```
[the remark this answers is not in the document]
Comment (IrvingJVR-MX) [reply src/components/Message.tsx]: done

**Review by gaguayovelez — APPROVED** [review]

**Comment by gaguayovelez** [comment]
@IrvingJVR-MX looks good to merge

**Pull request merged**